content/200-concepts/150-data-platform/03-projects/800-upgrading-plans.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Upgrading plans'
+metaTitle: 'Upgrading plans'
+metaDescription: 'You can upgrade to a paid plan to increase the Prisma Data Proxy usage and the number of team members and environments that you can add to your project.'
+---
+
+<TopBlock>
+
+Each project you create in the Prisma Data Platform starts by running on a free plan (`Early Adopter` or `Free`).
```

Comment:
We are going to remove the option to create Early Adopter plans on the day of the GA release. I am wondering if this could be confusing if read after that.

Perhaps we can phrase this differently? e.g. call out the fact there may also be "Early Adopter" projects from before GA?